### PR TITLE
quick refactor of userEncryption and comment alteration on some files

### PR DESCRIPTION
**Pull request by snowjtSD**
I made some small changes to the userEncryption.java file to meet the google documentation we are operating under. I also just made small comment changes to some of the files as well

### CODE REVIEW

File: app/src/main/java/com/yagni/model/Update.java [hunk 1]
```diff
@@ -4,72 +4,76 @@
...
 public class Update {
-	
-	private static String updateID, updateSupplierID, updateQuant, updatePrice, updateWholesale;
 
-	//Constructor
-    public Update(String productIdIn, String quantityIn, String wholeSaleIn, String salePriceIn, String supplierIdIn){
-        updateID = productIdIn;
+	// String Variables declaration
+	private static String updateID;
+	private static String updateSupplierID;
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/Update.java:10]: thanks for breaking these declarations up

File: app/src/main/java/com/yagni/model/UserEncryption.java [hunk 2]
```diff
@@ -2,21 +2,24 @@
 
 public class UserEncryption {
 
-    private static String hashedVal;
+    private static String hashedVal; // String declaration
+
+    private static char[] charVals; // char array declaration
```
Comment (S4kRiPp4) [inline app/src/main/java/com/yagni/model/UserEncryption.java:7]: nice addition w/ declaration

**Review by S4kRiPp4 — APPROVED** [review]
Seen you make corrections live, thanks for changes, good eye.

**Review by Jesusxx-240 — APPROVED** [review]
Good changes, I know the proper formatting style can be confusing be glad to see it correctly done now.

**Pull request merged**